resources/knative-eventing/charts/knative-eventing/Chart.yaml

```diff
@@ -1,7 +1,6 @@
-apiVersion: v1
+apiVersion: v2
 name: knative-eventing
 version: 0.12.0
```

Comment:
version -> this is the Chart version
appVersion -> this is the version of the application
In this case, `0.12.0` should be the appVersion, IMO

Reply:
I didn't change the version, it seems that it is the Knative Eventing version used.